Spec/Auth.swift

```diff
@@ -715,6 +715,32 @@ class Auth : QuickSpec {
             }
         }
 
+        // RSA8f3
+        pending("ensure the message published with a wildcard '*' does not have a clientId") {
+            let token = getTestToken()
+            let options = ARTClientOptions(token: token)
+            options.environment = "sandbox"
+            options.clientId = "john"
```

Comment:
See PR for RSA8f4, this is wrong, you need a token with a wildcard clientId and clientId is not set in the options.